ui-kit/src/main/res/layout/widget_send_details_input.xml

```diff
@@ -41,6 +41,7 @@
         app:layout_constraintHorizontal_bias="1"
         app:layout_constraintStart_toEndOf="@id/textViewSendLabel"
         app:layout_constraintTop_toTopOf="parent"
+        app:layout_goneMarginEnd="32dp"
```

Comment:
marginEnd or goneMarginEnd ?

Reply:
goneMarginEnd it is for cases when we make progressBar  GONE but we do not want to shift this view